bots/c/client_lib/src/internal/grpc_bridge.cpp

```
@@ -0,0 +1,539 @@
+	auto ctx = make_ctx();
+	grpc::Status status = g_stub->GetMyUnits(ctx.get(), req, &resp);
+	if (!status.ok()) return nullptr;
+
+	int count = resp.units_size();
+	if (count == 0) return nullptr;
+
+	t_obj **result = (t_obj **)malloc(sizeof(t_obj *) * (count + 1));
+	for (int i = 0; i < count; i++)
+	{
+		t_obj obj = proto_to_obj(resp.units(i));
+		result[i] = cache_store(obj);
```

Cppcheck (2.19.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)